File: src/main/java/com/example/examplemod/ExamplePlugin.java
package com.example.examplemod;

import emu.lunarcore.plugin.Plugin;
import org.slf4j.Logger;

import java.io.File;
import java.net.URLClassLoader;

@SuppressWarnings("unused")
public class ExamplePlugin extends Plugin {
    public ExamplePlugin(Identifier identifier, URLClassLoader classLoader, File dataFolder, Logger logger) {
        super(identifier, classLoader, dataFolder, logger);
    }

    @Override
    public void onEnable() {
    }

    @Override
    public void onDisable() {
    }
}

File: src/main/kotlin/com/example/examplemod/ExamplePlugin.kt
package com.example.examplemod

import emu.lunarcore.plugin.Plugin
import org.slf4j.Logger
import java.io.File
import java.net.URLClassLoader

@Suppress("unused")
class ExamplePlugin(identifier: Identifier, classLoader: URLClassLoader, dataFolder: File, logger: Logger) : Plugin(identifier, classLoader, dataFolder, logger) {
    override fun onEnable() {
    }

    override fun onDisable() {
    }
}